src/runtime.js

```diff
@@ -67,6 +67,22 @@ setReportError(reportErrorForWin.bind(null, self));
 /** @const @private {string} */
 const TAG = 'runtime';
 
+/**
+ * @typedef {{
```

Comment:
The same question here: is this a typedef or a bag of exported symbols? Obfuscation rules could differ between the two.

Reply:
We definitely don't want obfuscation here, so what's the correct way to do this?

Reply:
Ping @erwinmombay to confirm.